fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -39,7 +44,7 @@ import org.mozilla.fenix.utils.Settings
 private const val CFR_TO_ANCHOR_VERTICAL_PADDING = -6
 
 /**
- * The minimum number of opened tabs to show the CFR.
+ * The minimum number of opened tabs to show the Total Cookie Protection CFR.
  */
 private const val CRF_MINIMUM_NUMBER_OPENED_TABS = 5
```

Comment:
optional typo to fix
```suggestion
private const val CFR_MINIMUM_NUMBER_OPENED_TABS = 5
```